features/all-positions/AllLiquidations.tsx

```diff
@@ -211,113 +242,144 @@ const AllLiquidations = () => {
+                    <br />
+                    Status{" "}
+                  </strong>
+                </TableCell>
+                <TableCell align="right">
+                  <strong>
+                    <SortableTableColumnHeader
+                      sortField={SORT_FIELD.LOCKED_COLLATERAL}
                     >
-                      {prettyAddress(liquidation.liquidationReceipt)}
-                    </a>
-                  </TableCell>
+                      Locked
+                      <br /> Collateral
+                      <br />({collSymbol}){" "}
+                    </SortableTableColumnHeader>
+                  </strong>
+                </TableCell>
+                <TableCell align="right" style={{ width: "100%" }}>
+                  <strong>
+                    <SortableTableColumnHeader
+                      sortField={SORT_FIELD.TOKENS_LIQUIDATED}
+                    >
+                      Liquidated <br />
+                      Synthetics
+                      <br />({tokenSymbol}){" "}
+                    </SortableTableColumnHeader>
+                  </strong>
+                </TableCell>
+                <Tooltip
+                  title={`If the index price at the liquidation timestamp was ${
+                    invertDisputablePrice ? `above` : `below`
+                  } this, then the liquidation would be disputable.`}
+                  placement="top"
+                >
                   <TableCell align="right">
-                    {translateLiquidationStatus(
-                      Number(liquidation.liquidationTimestamp),
-                      liquidation.status
-                    )}
+                    <strong>
+                      <SortableTableColumnHeader
+                        sortField={SORT_FIELD.MAX_DISPUTABLE_PRICE}
+                      >
+                        {invertDisputablePrice ? `Minimum` : `Maximum`}{" "}
+                        Disputable Price{" "}
+                      </SortableTableColumnHeader>
+                    </strong>
                   </TableCell>
-                </TableRow>
-              );
-            })}
-          </TableBody>
-        </Table>
-        <PageButtons>
-          <div
-            onClick={() => {
-              setPage(page === 1 ? page : page - 1);
-            }}
-          >
-            <Arrow faded={page === 1 ? true : false}>←</Arrow>
-          </div>
-          {"Page " + page + " of " + maxPage}
-          <div
-            onClick={() => {
-              setPage(page === maxPage ? page : page + 1);
-            }}
-          >
-            <Arrow faded={page === maxPage ? true : false}>→</Arrow>
-          </div>
-        </PageButtons>
-      </TableContainer>
+                </Tooltip>
+
+                <TableCell align="right"></TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {reformattedLiquidationData.map((sponsor: string) => {
```

Comment:
I think you need to merge in `master` as the keys for `reformattedLiquidationData` are now `sponsorPlusId`